Kickstarter-iOS/Views/Controllers/CommentsViewController.swift

```diff
@@ -129,10 +129,14 @@ internal final class CommentsViewController: UITableViewController {
         self?.tableView.reloadData()
       }
 
-    self.viewModel.outputs.goToCommentReplies
+    self.viewModel.outputs.goToRepliesWithCommentProjectAndShowKeyboard
```

Comment:
Let's rename this and other instances of `showKeyboard` to be `becomeFirstResponder` for consistency:

`goToRepliesWithCommentProjectAndBecomeFirstResponder`.